docs/pages/documentation/3-querying-with-graql/delete-queries.md

```diff
@@ -23,6 +23,7 @@ To follow along, or experiment further, with the examples given below, please lo
 <div role="tabpanel" class="tab-pane active" id="shell1">
 <pre>
 match $x isa person; delete $x;
+commit
```

Comment:
I think we don't commit here on purpose because it affects our internal docs tests.

Reply:
I have removed this